system-contract-dapp-playground/README.md

```diff
@@ -98,6 +98,27 @@ npm install
 
 Please follow this [instruction here](https://support.metamask.io/hc/en-us/articles/360015489331-How-to-import-an-account) on how to import your account to Metamask using your **_HEX Encoded Private Key_** found in your Hedera Portal profile.
 
+## Project Prerequisite Check for Contract Assets
+
+**_note_**: The DApp Playground requires smart contract metadata in the root `/hedera-smart-contracts project` to retrieve bytecode for deployment and ABI for decoding details. To ensure all the necessary smart contract assets are up-to-date, please follow the steps below:
+
+1.  Follow these [guidelines](https://github.com/hashgraph/hedera-smart-contracts/blob/main/TEST_SETUP.md#building-and-running-the-project) to set up the environment variables and compile the smart contracts in the root directory.
```

Comment:
Might be more than people need. 
SHouldn't it be sufficient to just do the following?
```
npm install
npx hardhat compile
```

If so, let's do that instead of sending readers to set env configs that are only needed when running the automated tests

Reply:
Oh people would need to set up .env file for the `hardhat` to run compile unfortunately.

Reply:
We should fix this. I've opened #293 
You can address that in a separate PR